src/PodActions.jsx

```diff
@@ -176,33 +156,32 @@ export class PodActions extends React.Component {
                                               this.props.onAddNotification({ type: 'danger', error, errorDetail: ex.message });
                                           })}
                               component="button">
-                    {_("Pause")}
-                </DropdownItem>,
-            );
-        }
+                {_("Pause")}
+            </DropdownItem>,
+        );
+    }
 
-        if (dropdownItems.length > 1) {
-            dropdownItems.push(<DropdownSeparator key="separator-1" />);
-        }
-        dropdownItems.push(
-            <DropdownItem key="action-delete"
+    if (dropdownItems.length > 1) {
+        dropdownItems.push(<DropdownSeparator key="separator-1" />);
+    }
+    dropdownItems.push(
+        <DropdownItem key="action-delete"
                           className="pod-action-delete pf-m-danger"
                           onClick={() => {
                               Dialogs.show(<PodDeleteModal pod={pod} />);
                           }}
                           component="button">
-                {_("Delete")}
-            </DropdownItem>,
-        );
+            {_("Delete")}
+        </DropdownItem>,
+    );
 
-        if (!dropdownItems.length)
-            return null;
+    if (!dropdownItems.length)
+        return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1146-20221123-171443-906cf86c-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)